### DISCUSSION TITLE
Two questions about 3ML

### DISCUSSION THREAD

**Discussion by changxuezhao** [threeML/threeML#585] [Q&A] [answered]
Dear Dr. Burgess,
I have two questions for you about 3ML. First of all, the statistical (whether chi2 or pgstat or BIC) obtained using XSPEC will be smaller than 3ML, I think this is the reason for the likelihood, my question is why the likelihood of 3ML is larger than other tools（xspec，rmfit）. Second, why can only the line graph of vFv be drawn without data points, and how to add data points to the vFv graph. Thanks and looking forward to your reply。

**Comment by grburgess** [comment]
Hi, 

I am not sure what you mean by the first point.

To the second point, there are no such thing as data points in vFv or any photon model space. Data points live in instrument counts space for example or some other space completely. Photon models must be convolved with responses into count space. The inverse is not possible. When vFv data points are plotted, they assume a model and are made by adhoc assumptions. Thus, we specifically do not plot them in 3ML because it is incorrect to do so.

**Reply by changxuezhao** [reply]
HI, thanks for your reply. For my first question, for example, I used 3ml to fit a time-integrated spectral of around 70s, and got an optimal model(such as Band) after comparing the BIC of several empirical models, however, the BIC for this best model is as high as around 10,000. At the same time, I use XSPEC to fit the same energy spectrum. Of course, the best model is still Band, but the BIC at this time is only about 1500. 3ML and xspec will get similar fitting results, however their BIC are quite different, I wonder what could be causing this. Thanks and looking forward to your reply!

**Comment by grburgess** [comment, marked as answer]
This appears to be an issue and not a discussion. I have transferred it to issues.
